suites/pacific/cephadm/tier-0_cephadm.yaml

```diff
@@ -19,6 +19,8 @@ tests:
       desc: installation of ceph pre-requisites
       module: install_prereq.py
       abort-on-fail: true
+      config:
+        is_production: true
```

Comment:
@manasagowri Is this suite runs against live repo, Can you please let me know why **is_production** set for this suite?

Reply:
Removed it